html/openstef_dashboard_doc.html

```diff
@@ -0,0 +1,53 @@
+<!--
```

Comment:
Bij mij (edge) is het tekstvak niet zo goed zichtbaar doordat er een link overheen wordt weergegeven. Misschien dat dit is verholpen door de positie van het vak naar rechtsboven aan te passen? Denk dat daar ook nog wel ruimte is:
![image](https://user-images.githubusercontent.com/18208480/147217442-c642ff18-c02c-4278-84cd-1614f4246f82.png)